lib/compress/zstd_compress.c

```diff
@@ -488,6 +551,9 @@ size_t ZSTD_CCtx_setParametersUsingCCtxParams(
                     cctx, ZSTD_p_overlapSizeLog, params->overlapSizeLog) );
     }
 
```

Comment:
Unrelated : I forgot to mention, I like the new function name.
It respects very well repository's naming convention.